STATUS_GO_VERSION

```diff
@@ -1 +1 @@
-0.23.0-beta.8-chaos
+190328-172624-dc2531
```

Comment:
I'm putting this PR on hold until we will release new status-go version. 

@gravityblast do you think we can use this guide to make new release? https://github.com/status-im/status-go/blob/develop/RELEASING.md

Reply:
I guess we could just switch to 0.23.0-beta.10-chaos release

Reply:
I'm not sure if the next release will continue from that branch or not, so I would wait for tomorrow for this